Aptoide/aptoide-client-v8
# [DT-767] SL NEW SDK

**What does this PR do?**

This PR aims at changing the SL aar version. We also had to send a payment partner attribute in case it is an AppCoins app to be able to identify the installs of apps that with AppCoins.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AptoideTaskInfoRepository.kt

**How should this be manually tested?**

Perform an install, check if everything is ok. If you want you can debug the bundle and you will see this attribute is added to apps that have appcoins integration

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-767](https://aptoide.atlassian.net/browse/DT-767)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
